pkg/k8s/portforward.go

```diff
@@ -180,11 +180,25 @@ func (pf *PortForward) URLFor(path string) string {
 	return fmt.Sprintf("http://127.0.0.1:%d%s", pf.localPort, path)
 }
 
-// getLocalPort binds to a free ephemeral port and returns the port number.
+// getLocalPort is used by dashboard.go to select a port for the dashboard.
+// It first checks the availability of the default port, defined in addr.
+// If that port is taken, it binds to a free ephemeral port and returns the
+// port number.
 func getLocalPort() (int, error) {
-	ln, err := net.Listen("tcp", ":0")
+	addr := "127.0.0.1:60606"
```

Comment:
TIOLI: how about port `50750`, which translates to 50L5D in pager code, which translates to 50 Linkerd? #DreamOfThe90s

Reply:
✅